files/entrypoint.sh

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env bash
+
+
+# Variables
+# --
+#
+# These values can be overridden by ENV Vars
+#
+K8S_VERSION=${K8S_VERSION:-v1.6.1}
+KUBEADM_RELEASE=${KUBEADM_RELEASE:-v1.6.0-alpha.0.2074+a092d8e0f95f52}
+CNI_RELEASE=${CNI_RELEASE:-07a8a28637e97b22eb8dfe710eeae1344f69d16e}
+
+ARCH=${ARCH:-amd64}
+CNI_BIN_DIR=${CNI_BIN_DIR:-/opt/cni}
+ROOTFS=${ROOTFS:-/rootfs}
+
+
+# Kubernetes Google Cloud Storage
+# --
+#
+# Handling usage of Dev / Final release buckets
+#
+K8S_URL=${K8S_URL:-https://storage.googleapis.com/kubernetes-release/release}
+CNI_URL=${CNI_URL:-https://storage.googleapis.com/kubernetes-release/network-plugins}
+
+echo "Installing kubeadm version: ${KUBEADM_RELEASE}"
+if [[ -z "${KUBEADM_URL}" ]]; then
+  if [[ -n "${KUBEADM_VERSION/v[0-9].[0-9].[0-9]/}" ]]; then
+    echo "Kubeadm: is a Git version"
+    KUBEADM_URL=https://storage.googleapis.com/kubernetes-release-dev/ci-cross
+  else
+    echo "Kubeadm: is a Release version"
+    KUBEADM_URL=https://storage.googleapis.com/kubernetes-release/release
+  fi
+fi
+echo "Kubeadm: Download url - ${KUBEADM_URL}"
+
+
+# Linux Distro
+# --
+#
+# Environment settings according to Distro
+#
+if [[ $1 == "coreos" ]]; then
+  BIN_DIR=${BIN_DIR:-/opt/bin}
+  KUBELET_EXEC=${KUBELET_EXEC:-/usr/lib/coreos/kubelet-wrapper}
+  EXTRA_ENVIRONMENT=${EXTRA_ENVIRONMENT:-"RKT_RUN_ARGS=--volume opt-cni,kind=host,source=/opt/cni --mount volume=opt-cni,target=/opt/cni --volume etc-cni,kind=host,source=/etc/cni --mount volume=etc-cni,target=/etc/cni"}
+elif [[ $1 == "ubuntu" || $1 == "debian" || $1 == "fedora" || $1 == "centos" ]]; then
+  BIN_DIR=${BIN_DIR:-/usr/bin}
+  KUBELET_EXEC=${KUBELET_EXEC:-${BIN_DIR}/kubelet}
+  EXTRA_ENVIRONMENT=${EXTRA_ENVIRONMENT:-"NOOP=true"}
+  INSTALL_KUBELET=1
+else
+  cat <<-EOF
+  Hi, you should run this container like this:
+  docker run -it -v /etc/cni:/etc/cni -v /etc/systemd:/etc/systemd -v /opt:/opt -v /usr/bin:/usr/bin luxas/kubeadm-installer your_os_here
```

Comment:
Yeah, I think that mounting `/etc:/rootfs/etc` etc like the uninstall command below is the right way to do it